openstack/compute/v2/flavors/requests.go

```diff
@@ -21,22 +21,12 @@ type ListOptsBuilder interface {
 //
 // If no IsPublic argument is supplied in the request, OpenStack treats the FlavorType as Project.
 type FlavorType string
+
 const (
-        Project FlavorType =  "true"
-        Private FlavorType = "false"
-        All FlavorType = "None"
+	Project FlavorType = "true"
```

Comment:
Ah, I see what you are saying now. One would never actually refer to the type name in practice, only the constant name and that's what will end up being used in code. I'm sorry.

I think @mfranc213's change of `FlavorType` to `FlavorAccess` works well with your original request:

```go
listOpts := flavors.listOpts{
  FlavorAccess: flavors.TypePublic,
}
```

It seems to fit well with how it's described in the [documentation](https://github.com/openstack/python-novaclient/blob/stable/newton/novaclient/v2/flavors.py#L101-L103). 

yes/no?